app/js/components/manageApps/ManageApps.jsx

```diff
@@ -8,11 +8,14 @@
  */
 import React from 'react';
 import axios from 'axios';
+import { Router, Route, Link, IndexRoute, hashHistory, browserHistory } from 'react-router';
 import AddAddon from '../manageApps/AddAddon.jsx';
-import { ApiHelper } from '../../../helpers/apiHelper';
-import { AddonList } from './AddonList';
+import BreadCrumbComponent from '../breadCrumb/BreadCrumbComponent.jsx';
+import Header from '../common/Header';
```

Comment:
You may want to adda file extension for this line.

Reply:
The file is a .js file so there's no need to add the extension. I added the extensions for all .jsx files

Reply:
Ditto